dsa-23au/java-dsa/jpract-app/src/test/java/com/practice/app/mergeTest.java

```diff
@@ -0,0 +1,35 @@
+package test.java.com.practice.app;
+
+
+import java.util.Random;
+
+import main.java.com.practice.app.mergeSort;
+
+public class mergeTest 
+{
+    public static void main(String[] args)
+{
+    int[] num = new int[10];
+    Random random = new Random();
+
+    int i;
+    int j;
+
+
+    for(i = 0; i < num.length; i++)
+    {
+        num[i] = random.nextInt(100000);
+        System.out.println(num[i]);
+
+    }
+
```

Comment:
line 26 makes a new MergeSort class instance, but the class declaration 'mergeSort' should be uppercase/PascalCase and in its own file.